import_dataset/schema_less_guide.md

```diff
@@ -0,0 +1,305 @@
+
+    	statType: dcs:measuredValue
+
+2. They should have well-formed, stable DCIDs.
+* As we incrementally add schema, we should not change this DCID.
+3. They should have a name.
+* We will not auto-generate names, so a user-provided name is essential for display on SV hierarchy, SV explorer, etc.
+* See more on naming in the [DCID Naming Conventions](#dcidnamingconventions) section.
+4. They can have additional constraints that refer to existing schema (as long as they satisfy #1).
+5. They should ideally be organized in a hierarchy made of SVGroups (especially if you add a pile of schema-less SVs).
+* The SVGroups should not have a “dc/g/” DCID prefix.
+* SVGroups are organized in a `specializationOf` hierarchy
+* The root of your hierarchy should be linked to a vertical SVGroup from [here](https://github.com/datacommonsorg/schema/blob/main/stat_vars/vertical_stat_var_groups.mcf)
+* Schema-less stat-vars should be linked to an SVGroup via `memberOf` property
+* Example: [/dc/g/Person_Gender](https://datacommons.org/browser/dc/g/Person_Gender)
+
+
+## <a name="datasettypes"></a>Dataset Types
+
+For each new data import, summarize the incoming dataset and identify the types of data contained within it with the following steps:
+
+
+
+1. Identify the source of the data: Determine where the data is coming from and what organization or agency is responsible for collecting and maintaining it. This information can be found in the metadata or documentation that accompanies the dataset.
+2. Review the dataset's structure: Look at the number of columns and rows, as well as the variable names and data types. This will give you an idea of the overall structure of the data and the types of variables it contains.
+3. Check the data quality: Look for missing data, outliers, or other issues that may impact the accuracy or completeness of the data. If possible, remove or correct any problematic data before importing it.
+4. Identify the types of data: Look at the variables and their values to determine what types of data the dataset contains. Some examples of data types include demographic information, economic indicators, and geographic data.
+
+
+## <a name="contributingnewvariables"></a>Contributing New Variables
+
+When importing new data into DC, there may be some existing variables from previous data imports that represent all or part of your new import. It is important to consider whether existing statistical variables can be reused or if new ones need to be created.
+
+Here is a quick 1-2 step guide to help you determine when to reuse existing variables and when to create new ones:
+
+
+
+1. **Check existing variables**: Before creating a new variable, take a look at our [Statistical Variable Explorer](https://www.datacommons.org/tools/statvar) to check if an existing variable can be used to represent the data you are importing. This will save time and effort, and it also helps to maintain consistency in the data commons.
```

Comment:
instead of "in the data commons" -- how about "in the graph", "in the data commons graph" or "on data commons"